inventory.lua

```diff
@@ -310,6 +314,68 @@ minetest.register_node("digilines:chest", {
 	end
 })
 
+if minetest.global_exists("tubelib") and minetest.global_exists("tubelib2") then
```

Comment:
Should this `or` instead of `and`?

Reply:
No, it's confusing. Tubelib2 is the base mod, it provides generic tube functionality. Tubelib is a part of techpack and is a specific implementation of tubelib2 for moving items around chests and machines and stuff.
I'm checking for both globals because I use them both, `tubelib` to register the chest as an item-receiving node, and `tubelib2` to avoid doing any clever side-to-facedir logic myself.
Tubelib depends on tubelib2, but I'd rather check for both just in case.

Reply:
That is very confusing... :/

Reply:
It would be less confusing if you'd only check for `tubelib` if that one's guaranteed to depend on `tubelib2`

Reply:
Alright, I'm just checking for tubelib now, though I still think it makes more sense to check for all the globals you use just to be safe